usb/backend/libusb1.py

```diff
@@ -731,6 +731,13 @@ def get_device_descriptor(self, dev):
         except AttributeError:
             dev_desc.port_numbers = None
 
+        try:
+            dev_desc.speed = self.lib.libusb_get_device_speed(dev.devid)
+            dev_desc.iSpeed = None
+        except Exception:
```

Comment:
Why the exception handler here? As the call to `libusb_get_device_speed` is not wrapped by `_check`, no exception should be raised.